components/Input/__test__/input.spec.ts

```diff
@@ -20,6 +20,51 @@ afterEach(() => {
 });
 
 describe('Test: KInput', () => {
+	test('props: input sm size', async () => {
+		const instance = new KInput({
+			target: host,
+			props: {
+				size: 'sm',
+				iconPrefix: 'i-prefix',
+				iconSuffix: 'i-suffix'
+			}
+		});
+		expect(instance).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--sm')).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--icon--sm')).toBeTruthy();
+		expect(host.innerHTML).matchSnapshot();
+	});
+
+	test('props: input md size', async () => {
+		const instance = new KInput({
+			target: host,
+			props: {
+				size: 'md',
+				iconPrefix: 'i-prefix',
+				iconSuffix: 'i-suffix'
+			}
+		});
+		expect(instance).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--md')).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--icon--md')).toBeTruthy();
+		expect(host.innerHTML).matchSnapshot();
+	});
+
+	test('props: input lg size', async () => {
+		const instance = new KInput({
+			target: host,
+			props: {
+				size: 'lg',
+				iconPrefix: 'i-prefix',
+				iconSuffix: 'i-suffix'
+			}
+		});
+		expect(instance).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--lg')).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--icon--lg')).toBeTruthy();
+		expect(host.innerHTML).matchSnapshot();
+	});
+
```

Comment:
Maybe it would be better to combine all size tests into one test. ['sm', 'md', 'lg'].forEach(s => {})